### PR TITLE
item_db and item_combo_db correction #5

### PR DESCRIPTION
**Pull request by kaninhot004**
* **Server Mode**: 

Renewal

* **Description of Pull Request**: 

Item DB & Item Combo DB updates.
Follow by lastest item info from kRO and Divine Pride with kRO server selected. (Some missing from jRO because it's not had on kRO)

Item DB

> 1322 - Correct up bonus and item data (https://www.divine-pride.net/database/item/1322/)
> 1336 - Added missing skill damage bonus (https://www.divine-pride.net/database/item/1336/)
> 1424 - Added missing large size damage bonus (https://www.divine-pride.net/database/item/1424/)
> 1426 - Added missing aspd rate bonus (https://www.divine-pride.net/database/item/1426/)
> 1435 - Changed atk bonus to skill damage bonus (https://www.divine-pride.net/database/item/1435/)
> 1437 - Adjusted curse rates (https://www.divine-pride.net/database/item/1437/)
> 1441 - Correct up bonus and item data (https://www.divine-pride.net/database/item/1441/)
> 1443 - Seperated atk bonus from new condition info (https://www.divine-pride.net/database/item/1443/)
> 1445 - Implemented missing items (https://www.divine-pride.net/database/item/1445/)
> 1448 - Implemented missing items (https://www.divine-pride.net/database/item/1448/)
> 1466 - Changed critical to critical rates (https://www.divine-pride.net/database/item/1466/)
> 1470 - Removed boss damage bonus (https://www.divine-pride.net/database/item/1470/)
> 1476 - Changed critical to critical rates (https://www.divine-pride.net/database/item/1476/)
> 1488 - Adjusted damage rates (https://www.divine-pride.net/database/item/1488/)
> 1489 - Removed unbreakable bonus (https://www.divine-pride.net/database/item/1489/)
> 1492 - Seperated magic damage return bonus from new condition info (https://www.divine-pride.net/database/item/1492/)
> 1494 - Added missing max hp rates bonus (https://www.divine-pride.net/database/item/1494/)
> 1495 - Adjusted blind rates (https://www.divine-pride.net/database/item/1495/)
> 1497 - Adjusted auto apwll skill level bonus (https://www.divine-pride.net/database/item/1497/)
> 1498 - Seperated atk bonus from new condition info (https://www.divine-pride.net/database/item/1498/)
> 1523 - Changed critical to critical rate and adjusted def decrease rates (https://www.divine-pride.net/database/item/1523/)
> 1541 - Changed race undead to element dark damage bonus (https://www.divine-pride.net/database/item/1541/)
> 1548 - Changed critical to critical rate bonus (https://www.divine-pride.net/database/item/1548/)
> 1567 - Remove mAtkRate bonus (https://www.divine-pride.net/database/item/1567/)
> 1573 - Added missing item bonus (https://www.divine-pride.net/database/item/1573/)
> 1574 - Correct up bonus and item data (https://www.divine-pride.net/database/item/1574/)
> 1575 - Correct up bonus (https://www.divine-pride.net/database/item/1575/)
> 1576 - Correct up bonus (https://www.divine-pride.net/database/item/1576/)
> 1577 - Correct up bonus (https://www.divine-pride.net/database/item/1577/)
> 1578 - Adjusted mDef bonus (https://www.divine-pride.net/database/item/1578/)
> 1580 - Removed mAtkRate bonus (https://www.divine-pride.net/database/item/1580/)
> 1585 - Change mAtk to atk bonus (https://www.divine-pride.net/database/item/1585/)
> 1586 - Correct up bonus (https://www.divine-pride.net/database/item/1586/)
> 1588 - Removed unbreakable bonus (https://www.divine-pride.net/database/item/1588/)
> 1590 - Added missing item bonus (https://www.divine-pride.net/database/item/1590/)
> 1591 - Change recovery bonus to heal power bonus (https://www.divine-pride.net/database/item/1591/)
> 1592 - Implemented missing items (https://www.divine-pride.net/database/item/1592/)
> 1593 - Implemented missing items (https://www.divine-pride.net/database/item/1593/)
> 1594 - Implemented missing items (https://www.divine-pride.net/database/item/1594/)
> 1595 - Implemented missing items (https://www.divine-pride.net/database/item/1595/)
> 1596 - Added missing int bonus (https://www.divine-pride.net/database/item/1596/)
> 1597 - Adjusted str bonus and remove unbreakble bonus (https://www.divine-pride.net/database/item/1597/)
> 1598 - Implemented missing items (https://www.divine-pride.net/database/item/1598/)
> 1599 - Added missing aspd rate bonus (https://www.divine-pride.net/database/item/1599/)
> 1664 - Added missing ignore mDefRate bonus (https://www.divine-pride.net/database/item/1664/)
> 1698 - Adjusted jobs req. (https://www.divine-pride.net/database/item/1698/)
> 29176 - Implemented missing items (https://www.divine-pride.net/database/item/29176/)
> 32234 - Implemented missing items (https://www.divine-pride.net/database/item/32234/)

Item Combo DB

> - RWC Staff, RWC Pendent, Chamber RWC Pendate use mAtk instead of mAtkRate
> - Added Snake Encyclopedia combo
> - Added Warrior's combo

Thanks to 🅺🅸🅽🅶🅰🆁🆃🅴🆁 # 4210 (Discord)

### CODE REVIEW

**Comment by kaninhot004** [comment]
Today I had more update. (Follow kRO description in Divine Pride)

**Comment by Atemo** [comment]
Anything left? If the work is still in progress, change the PR to draft

**Comment by kaninhot004** [comment]
> Anything left? If the work is still in progress, change the PR to draft

It's done now. Thanks!

By the way next time I will keep it on draft, If still updating.

**Comment by KingarteR** [comment]
please check and merge to master

File: db/re/item_db.txt [hunk 1]
```diff
@@ -12706,6 +12727,7 @@
...
 28564,Valkyrie_Drop,Valkyrie Drop,4,20,,30,,0,,1,0xFFFFFFFF,63,2,136,,100,0,0,{ .@r = getskilllv("PR_IMPOSITIO"); bonus bSPrecovRate,50; bonus bVariableCastrate,-10; bonus bDelayrate,-5; bonus2 bVariableCastrate,"AB_ORATIO",-50; bonus2 bFixedCastrate,"AB_ORATIO",-100; bonus2 bMagicAddSize,Size_All,3*.@r; bonus2 bSkillAtk,"PR_MAGNUS",3*(Baselevel/2); if (.@r == 5) skill "ALL_ODINS_POWER",2; },{},{}
 28565,Perverse_Demon_Mask,Perverse Demon Mask,4,20,,60,,0,,1,0xFFFFFFFF,63,2,128,,80,0,0,{ .@s = readparam(bStr); .@a = readparam(bAgi); .@d = readparam(bDex); .@v = readparam(bvit); .@l = readparam(bLuk); .@i = readparam(bInt); bonus bStr,3*(.@i/18); bonus bAgi,3*(.@l/18); bonus bVit,3*(.@d/18); bonus bInt,3*(.@s/18); bonus bDex,3*(.@v/18); bonus bLuk,3*(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bFlee2,(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bVariableCastrate,-(.@v/18); bonus bDelayrate,-(.@i/18); bonus2 bIgnoreDefClassRate,Class_All,15*(.@l/18); bonus2 bIgnoreMdefClassRate,Class_All,15*(.@s/18); },{},{}
+28566,Beginner's_Ring,Beginner's Ring,4,20,,0,,0,,0,0xFFFFFFFF,63,2,128,,100,0,0,{ bonus bAspdRate,10; bonus bVariableCast,10; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:12730]:
```suggestion
28566,Beginner's_Ring,Beginner's Ring,4,20,,0,,0,,0,0xFFFFFFFF,63,2,128,,100,0,0,{ bonus bAspdRate,10; bonus bVariableCastrate,-10; },{},{}
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db.txt [hunk 2]
```diff
@@ -924,35 +926,40 @@
...
+1593,Book_of_Destiny,Book of Destiny,5,10,,400,90:90,,1,1,0x00410100,63,2,2,4,24,1,15,{ .@r = getrefine(); bonus bLuk,5; bonus bFlee2,15; if(.@r >= 7) bonus bFlee2,5; bonus bUnbreakableWeapon; },{},{}
+1594,Fire_Pedigree_Book,Fire Pedigree Book,5,10,,400,90:130,,1,2,0x00410100,63,2,2,4,27,1,15,{ .@r = getrefine(); bonus bInt,3; bonus2 bSkillAtk,"MG_FIREBOLT",5+.@r; bonus bUnbreakableWeapon; bonus bAtkEle,Ele_Fire; },{},{}
+1595,Water_Pedigree_Book,Water Pedigree Book,5,10,,400,90:130,,1,2,0x00410100,63,2,2,4,27,1,15,{ .@r = getrefine(); bonus bInt,3; bonus2 bSkillAtk,"MG_COLDBOLT",5+.@r; bonus bUnbreakableWeapon; bonus bAtkEle,Ele_Water; },{},{}
+1596,Earth_Pedigree_Book,Earth Pedigree Book,5,10,,400,90:130,,1,2,0x00410100,63,2,2,4,27,1,15,{ .@r = getrefine(); bonus bInt,3; bonus2 bSkillAtk,"WZ_HEAVENDRIVE",5+.@r; bonus2 bSkillAtk,"WZ_EARTHSPIKE",5+.@r; bonus bUnbreakableWeapon; },{},{}
+1597,Half_BF_Book2,Half BF Book2,5,20,,0,90:125,,1,0,0x00210100,63,2,2,3,80,1,15,{ bonus bStr,1; bonus bInt,1; bonus2 bAddRace,RC_DemiHuman,35; bonus2 bAddRace,RC_Player_Human,35; bonus2 bIgnoreMdefRaceRate,RC_DemiHuman,10; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,10; },{},{}
```
[the remark this answers is not in the document]
Comment (kaninhot004) [reply db/re/item_db.txt:959]: From Divine Pride
image

**Comment by kaninhot004** [comment]
All changes is done. Thank you!

File: db/re/item_db.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Atemo) [reply db/re/item_db.txt:959]: I mean I can't find 1597 on DP or aegis, where did you get it ?

**Comment by Atemo** [comment]
My comment were lost in the review, I can't find 1597 on DP or aegis, where did you get it ?

**Comment by kaninhot004** [comment]
> My comment were lost in the review, I can't find 1597 on DP or aegis, where did you get it ?

I selected kRO server on DP and it's show up.
https://www.divine-pride.net/database/item/1594

**Comment by Atemo** [comment]
Alright, we can keep it it's fine!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**